src/commands/update.ts

```diff
@@ -1,11 +1,14 @@
+import type { GuildMember, TextChannel } from "discord.js";
 import type { Command } from "../command";
 
 const command: Command = {
 	name: "update",
 	description: "Shutdowns the bot for an update.",
 	options: [],
-	hasPermission: (state, message) => !!message.member?.roles.cache.has(state.config.developerRoleId),
-	parseArguments: () => [],
+	hasPermission: (state, interaction) =>
+		(interaction.member as GuildMember).roles.cache.has(state.config.developerRoleId),
+	shouldBeEphemeral: (state, interaction) =>
```

Comment:
Is this needed since the command does not notify the user at all? Should we notify the end user of it restarting?

Reply:
Yeah there's no point since there's no notification. We could notify the user to just say "Restarting..." but the bot going offline then back online sort of does that  for us.